src/Bicep.Core/TypeSystem/DeclaredTypeManager.cs

```diff
@@ -939,7 +942,14 @@ private DeclaredTypeAssignment GetModuleType(ModuleDeclarationSyntax syntax)
                 syntax.Value is ForSyntax ? new TypedArrayType(declaredModuleType, TypeSymbolValidationFlags.Default) : declaredModuleType,
                 syntax);
         }
+        private DeclaredTypeAssignment GetTestType(TestDeclarationSyntax syntax)
```

Comment:
[nit]: This applies throughout, but try to keep one blank line between each method declaration.